jest.config.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
`s/precompiler by the library author/precompiled by the library author/`? in the commit message